metagraph/src/graph/alignment/dbg_aligner.hpp

```diff
@@ -155,14 +155,14 @@ ::align(const std::string_view query,
     });
 
     for (auto &seed : seeds) {
-        mtg::common::logger->trace("Seed: {}", seed);
+        // mtg::common::logger->trace("Seed: {}", seed);
```

Comment:
?

Reply:
This creates too much output, so I disabled them for now. See above